scripts/jenkins/jenkinsfiles/Jenkinsfile-Release

```diff
@@ -0,0 +1,283 @@
+//                            echo \${list_of_css_files}
+//                            for f in \${list_of_css_files}
+//                            do
+//                                s3cmd --acl-public --mime-type text/css put target/\${f} ${S3_ROOT}/${env.BRANCH_NAME}/${currentBuild.number}/\${f}
+//                            done
+//
+//                            echo UPDATE LATEST POINTER
+//                            tmpdir=./buildh2odev.tmp
+//                            mkdir -p \${tmpdir}
+//                            echo ${currentBuild.number} > \${tmpdir}/latest
+//                            echo "<head>" > \${tmpdir}/latest.html
+//                            echo "<meta http-equiv=\\"refresh\\" content=\\"0; url=${currentBuild.number}/index.html\\" />" >> \${tmpdir}/latest.html
+//                            echo "</head>" >> \${tmpdir}/latest.html
+//
+//                            s3cmd --acl-public put \${tmpdir}/latest ${S3_ROOT}/${env.BRANCH_NAME}/latest
+//                            s3cmd --acl-public put \${tmpdir}/latest.html ${S3_ROOT}/${env.BRANCH_NAME}/latest.html
+//                            s3cmd --acl-public put \${tmpdir}/latest.html ${S3_ROOT}/${env.BRANCH_NAME}/index.html
+//                        """
+//                    }
+//                }
+
+                if (env.BRANCH_NAME.startsWith(buildConfig.RELEASE_BRANCH_PREFIX)) {
+                    if (params.UPLOAD_PYPI) {
+                        stage('Upload to PyPI') {
+                            insideDocker([], buildConfig.DEFAULT_IMAGE, buildConfig.DOCKER_REGISTRY, 2, 'HOURS', "-v ${env.BRANCH_DIR}:${env.BRANCH_DIR}") {
+                                withCredentials([usernamePassword(credentialsId: 'pypi-credentials', usernameVariable: 'TWINE_USERNAME', passwordVariable: 'TWINE_PASSWORD')]) {
+                                    sh 'printenv'
+                                    echo 'Upload to PyPI'
+                                    sh """
+                                        cd ${env.BUILD_NUMBER_DIR}/h2o/h2o-py
+                                        python setup.py sdist
+                                        
+                                        # FIXME
+                                        # twine upload dist/h2o-${PROJECT_VERSION}.tar.gz
+                                    """
+                                }
+                            }
+                        }
+                    }
+                    if (params.UPLOAD_CONDA) {
+                        stage('Upload to Conda') {
+                            withCredentials([usernamePassword(credentialsId: 'anaconda-credentials', usernameVariable: 'ANACONDA_USERNAME', passwordVariable: 'ANACONDA_PASSWORD')]) {
+                                sh 'printenv'
+                                echo 'Upload to Conda'
+                                sh """
+                                    cd ${env.BUILD_NUMBER_DIR}/h2o/h2o-py/conda
```

Comment:
This should be updated according to the latest h2o release script https://github.com/h2oai/h2o-release-scripts/blob/master/buildh2o3.sh as the code for releasing conda has changed since then